### ISSUE TITLE
Submission associated with some track are not visible

### ISSUE THREAD

**Issue by jcfr** [Kitware/CDash#459]
I just noticed that the Slicer submissions associated with the dashboard group "Nightly-Packages" are not visible anymore.

I suspect this is due to the recent update ...

Dashboard: http://slicer.cdash.org/index.php?project=Slicer4

Relevant part of the ctest script: https://github.com/Slicer/Slicer/blob/2a6e43ecd5bff6e235db298dc38354279381b9e7/CMake/SlicerDashboardDriverScript.cmake#L250

Here is part of the relevant log:

```
Directory /home/kitware/Dashboards/Nightly/Slicer-0-build cleaned !
SetCTestConfiguration:SourceDirectory:/home/kitware/Dashboards/Nightly/Slicer-0
SetCTestConfiguration:BuildDirectory:/home/kitware/Dashboards/Nightly/Slicer-0-build
Run dashboard with model Nightly
   Source directory: /home/kitware/Dashboards/Nightly/Slicer-0
   Build directory: /home/kitware/Dashboards/Nightly/Slicer-0-build
   Track: Nightly-Packages
   Reading ctest configuration file: /home/kitware/Dashboards/Nightly/Slicer-0/CTestConfig.cmake
SetCTestConfigurationFromCMakeVariable:NightlyStartTime:CTEST_NIGHTLY_START_TIME
SetCTestConfiguration:NightlyStartTime:3:00:00 UTC
SetCTestConfigurationFromCMakeVariable:Site:CTEST_SITE
SetCTestConfiguration:Site:factory-south-ubuntu-64bits.kitware
SetCTestConfigurationFromCMakeVariable:BuildName:CTEST_BUILD_NAME
SetCTestConfiguration:BuildName:Linux-g++4.4.3-64bits-QT4.8.6-NoVTKDebugLeaks-Doxygen-Release
   Site: factory-south-ubuntu-64bits.kitware
   Build name: Linux-g++4.4.3-64bits-QT4.8.6-NoVTKDebugLeaks-Doxygen-Release
Determine Nightly Start Time
   Specified time: 3:00:00 UTC
   Use Nightly-Packages tag: 20170106-0300
SetCTestConfiguration:BuildDirectory:/home/kitware/Dashboards/Nightly/Slicer-0-build
SetCTestConfiguration:SourceDirectory:/home/kitware/Dashboards/Nightly/Slicer-0
SetCTestConfiguration:SourceDirectory:/home/kitware/Dashboards/Nightly/Slicer-0
[...]
```

Here is a snipped of the Update.xml submitted last night:

```
<Update mode="Client" Generator="ctest-3.5.0-rc3">
        <Site>factory-south-ubuntu-64bits.kitware</Site>
        <BuildName>Linux-g++4.4.3-64bits-QT4.8.6-NoVTKDebugLeaks-Doxygen-Release</BuildName>
        <BuildStamp>20170106-0300-Nightly-Packages</BuildStamp>
        <StartDateTime>Jan 05 23:10 EST</StartDateTime>
        <StartTime>1483675821</StartTime>
[...]
```

Thanks for the help,

**Comment by jcfr** [comment]
Looking at the Update.xml of the last submission of overload dashboard, the `StartDateTime`:

```
<StartDateTime>Jan 08 22:11 Eastern Standard Time</StartDateTime>
```

**Comment by zackgalbreath** [comment]
Ok, I think I figured this out!  This problem occurs when submitting a Upload.xml file for a buildgroup whose name contains a hyphen.

The fix is here: #461.

**Comment by jcfr** [comment]
Great :+1: I confirm that #461 fixes the problem :fireworks:
